mathesar/api/exceptions/mixins.py

```diff
@@ -81,7 +81,7 @@ def _run_validator(self, validator, field, message):
                 args.append(field)
             validator(self.initial_data[field.field_name], *args)
         except (DjangoValidationError, RestValidationError) as err:
-            err_message = err.detail[0] if hasattr(err, 'detail') else err.message
+            err_message = err.detail[0] if hasattr(err, 'detail') else err.messages[0]
```

Comment:
- Django error objects sometimes contain only the `message` field. So `err.message` is sometimes valid too. So please revert its removal. In order to support various types of errors,  you would need to write the logic for each case.
- Also make sure to check if the `messages` array is not empty before fetching the first error object.

Reply:
Valid points @silentninja, Will implement the suggested changes.